app/views/settings/preferences/appearance/show.html.haml

```diff
@@ -56,6 +59,3 @@
 
   .fields-group
     = f.input :setting_expand_spoilers, as: :boolean, wrapper: :with_label
-
-  .actions
-    = f.button :button, t('generic.save_changes'), type: :submit
```

Comment:
Appearance settings page has many elements and long enough. I think it is a good idea to leave another button at the bottom too.

Reply:
You're right, I just added again the button at the bottom of the page